### PR TITLE
changes to make the Windows MD binaries discoverable by CMake

### PR DESCRIPTION
**Pull request by giumas**
https://github.com/conda-forge/gtest-feedstock/issues/16

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,11 +45,10 @@ test:
     - if not exist %LIBRARY_BIN%\gtest.lib exit 1           # [win]
     - if not exist %LIBRARY_BIN%\gtest_main.lib exit 1      # [win]
     - if not exist %LIBRARY_BIN%\gtest_dll.dll exit 1       # [win]
-    - if not exist %LIBRARY_BIN%\gtest_md.lib exit 1        # [win]
-    - if not exist %LIBRARY_BIN%\gtest_md.lib exit 1        # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml:49]: I'm guessing one of these was suppose to be `.dll`. Can we add such a test below for the new filename?

Comment (giumas) [reply recipe/meta.yaml:49]: Great catch!

**Comment by jakirkham** [comment]
LGTM

**Comment by SylvainCorlay** [comment]
@giumas This seems to have broken the tests for a bunch of downstream libraries that make use of gtest:

```
gtest-md.lib(gtest-all.cc.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MT_StaticRelease' in main.cpp.obj
```

**Comment by giumas** [comment]
The error seems clear. They are linking to `gtest-md.lib` while they should link to `gtest.lib`. 

Since `gtest-md.lib` was not there before, I don't understand how the changes can have broken depending libraries.  Can you point me to some examples?

**Comment by SylvainCorlay** [comment]
Thank you for replying so quickly! Indeed, I meant the changes of the previous PR...

In fact, the error comes from a workaround that we were doing with the earlier versions of the build. Now fixed by https://github.com/QuantStack/xtensor/pull/393.

**Pull request merged**